src/frontend/packages/core/assets/nav-logo-icon.png

```diff
@@ -0,0 +1 @@
+/Users/nwm/dev/nav2/src/frontend/packages/core/misc/custom/nav-logo-icon.png
```

Comment:
Don't think this should be included

Reply:
Yes - this is the default icon for when the side bar is icon-mode

Reply:
It's a symlink though to `src/frontend/packages/core/misc/custom/nav-logo-icon.png`, if i run customize it updates and the change is tracked by git

Reply:
For metrics I used bar_chart as the icon